.dockstore.yml

```diff
@@ -0,0 +1,6 @@
+# Dockstore config version, not pipeline version
+version: 1.2
+workflows:
+  - subclass: nfl
```

Comment:
It sould be good to add a name to the workflow
```suggestion
  - name: NPM-sample-qc
    subclass: nfl
```

Reply:
In this brnach the `name` and `author` details are given in the `nextflow.config`. Let me see how to include the email and orcid under the manifest.
https://github.com/c-BIG/NPM-sample-qc/blob/feature/metrics-fix/nextflow.config